websock/http/server.nim

```diff
@@ -162,7 +162,7 @@ proc handleTlsConnCb(
   finally:
     await stream.closeWait()
 
-proc accept*(server: HttpServer): Future[HttpRequest]
+proc accept*(server: HttpServer, timeout: Duration = 5.seconds): Future[HttpRequest]
```

Comment:
Accept shouldn't take the timeout, it's better to allow configuring it during object creation and we should call it something like `HandshakeTimeout`.

Reply:
resolved in a6ba0b9256ef47ba4423441949c01b2be270ea32